PersonPose[] personPoses
int32 numPerson
